modules/API/pages/built-in-endpoints.adoc

```diff
@@ -741,9 +741,15 @@ $ curl -X GET 'http://localhost:9000/deleted_vertex_check?threadnum=10&verbose=0
 
 The endpoints in this subsection allow users to create, refresh and delete authentication tokens for requests made to the REST{pp} server. *These endpoints only exist when* xref:user-access:enabling-user-authentication.adoc[*user authentication is enabled*] *on RESTPP endpoints.*
 
-=== Request a token (`GET`)
+[CAUTION]
+====
+Please be aware to pass parameters in the json payload for `/requesttoken` api from 3.5. If the old API version is used, it could cause information leakage.
```

Comment:
How about this?

```
[CAUTION]
====
Please be aware to pass parameters in the json payload for `/requesttoken` api from 3.5. If the API version before version 3.5 is used, it could cause information leakage.
====
```

Reply:
Hmm, from what I understand, this is to warn people not to use the `/requesttoken` API from 3.4 and prior? In that case, the warning perhaps should be in the older versions instead of here. If a user is reading the 3.5 documentation, we assume that they are using 3.5. 

Is the risk for information leakage high? This sounds a little scary if we say it this way, so I want to make sure we are communicating it appropriately.